gulpfile.js

```diff
@@ -308,7 +351,7 @@ const displayHelp = (cb) => {
 log('🪴 building the cmsds');
 exports.build = gulp.series(
   cleanDist,
-  gulp.parallel(copyThemes, copyImages, copyFonts, copyJSON),
+  gulp.parallel(copyCssThemes, copyScssThemes, copyThemesToDocs, copyImages, copyFonts, copyJSON),
```

Comment:
I've been getting an error when I try to build, and I can't figure out why:
```
[16:12:54] The following tasks did not complete: build, <parallel>, 🖍  compiling sass into css
[16:12:54] Did you forget to signal async completion?
```

All the functions look good to me. We're calling the callback functions as expected.

Reply:
lol i'm an idiot. it was caused by a typo in the updated exportCssVars - i was exporting a sass file called `{theme}-core-tokens.scss` instead of `${theme}-layout-tokens.scss`. pushing the fix up now

Reply:
oh wow, ha ha! I could *not* see it no matter how hard I looked